pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -208,6 +208,21 @@ func (sna *SnapshotAlpha) CreateFromSource(ctx context.Context, source *Source,
 	if !waitForReady {
 		return nil
 	}
+	if source.Driver == "csi-vxflexos.dellemc.com" {
```

Comment:
Minor - let's make this a constant.

Reply:
yep will do

Reply:
Add a comment with the link to the upstream PR here please - indicating this is a workaround till the upstream issue is addressed

Reply:
Add a unit test please?

Reply:
@vkamra UT'd